.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Release
+on:
+    push:
+        branches:
+            - master
+jobs:
+    release:
+        name: Build and Release
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
```

Do we need write permission for this? 

I guess read would be ok
```suggestion
            contents: read
```

---

This is needed to be able to publish a GitHub release